src/stream/src/executor/aggregation/mod.rs

```diff
@@ -423,7 +424,11 @@ pub fn generate_state_table<S: StateStore>(
             };
             relational_pk_len
         ],
-        None,
+        if dist_keys.is_empty() {
+            None
+        } else {
+            Some(dist_keys)
+        },
```

Comment:
Do the `if` branches correspond to simple agg and hash agg respectively? It would be better to add some comments.

Reply:
Yes, it's in mod.rs and used by simple agg/hash agg. Will add comments.

Actually I think the parameters should not be a Option, cuz `dist_keys.is_empty() true` means None.

Reply:
+1. We can remove the `Option` after most of the executors provide dist keys correctly.